### PR TITLE
Treeportlet: extend caching parameters with current language code.

### PR DESCRIPTION
**Pull request by phgross**
This allows us to display language dependent content in the Treeportlet. As we do for some customers.

@lukasgraf please have a look

### CODE REVIEW

File: opengever/base/utils.py [hunk 1]
```diff
@@ -64,3 +64,12 @@ def ok_response(request=None):
...
 def disable_edit_bar():
     api.portal.get().REQUEST.set('disable_border', True)
+
+
+def get_preferred_language_code():
+    ltool = api.portal.get_tool('portal_languages')
+    language_code = ltool.getPreferredLanguage()
+    # special handling for combined languages
+    if len(language_code.split('-')) > 1:
+        return language_code[0]
```
Comment (lukasgraf) [inline opengever/base/utils.py]: This should be `language_code.split('-')[0]` (or equivalent), no?

**Comment by lukasgraf** [comment]
Other than that: :+1:

**Comment by phgross** [comment]
@lukasgraf updated as we discussed ... 

`return language_code.split('-')[0]` works for both

**Comment by lukasgraf** [comment]
:+1:

**Pull request merged**